src/js.cookie.js

```diff
@@ -1,10 +1,10 @@
 /*!
- * JavaScript Cookie v2.1.0
- * https://github.com/js-cookie/js-cookie
- *
- * Copyright 2006, 2015 Klaus Hartl & Fagner Brack
- * Released under the MIT license
- */
+* JavaScript Cookie v2.1.0
+* https://github.com/js-cookie/js-cookie
+*
+* Copyright 2006, 2015 Klaus Hartl & Fagner Brack
+* Released under the MIT license
+*/
```

Comment:
I've used Atom to auto-indent the code, I'm not particular about spacing here though, will revert